cudl-staging/terraform.tfvars

```diff
@@ -20,8 +19,6 @@ lambda-db-jdbc-driver                = "org.postgresql.Driver"
 lambda-db-url                        = "jdbc:postgresql://<HOST>:<PORT>/staging_cudl_viewer?autoReconnect=true"
 lambda-db-secret-key                 = "staging/cudl/cudl_viewer_db"
 
-// NOTE: If you are adding anything here you need to add a code block to
-// the s3.tf file
 source-bucket-sns-notifications = [
```

Comment:
Would it make sense to rename this variable to `transform-lambda-bucket-sns-notifications` in the `cudl-data-processing` module as the existing name is confusing given the list can now also contain the dest bucket?

Reply:
Renamed.